proxy/config/records.config.default.in

```diff
@@ -97,7 +97,8 @@ CONFIG proxy.config.http.cache.http INT 1
 #    https://docs.trafficserver.apache.org/en/latest/admin-guide/files/cache.config.en.html
 ##############################################################################
 CONFIG proxy.config.http.cache.ignore_client_cc_max_age INT 1
-CONFIG proxy.config.http.normalize_ae_gzip INT 1
+CONFIG proxy.config.http.normalize_ae INT 0
```

Comment:
I think this should be '1' and `normalize_ae_gzip` should be removed entirely from the defaults.

Reply:
@SolidWallOfCode It seems like we would need a consensus of all the ops people using ATS that it's cool for a new version of ATS to potentially break or not work as before on their existing config files.  How do we get this consensus?

Reply:
Removing something from the default configuration will not break any production deployments. If I've read this PR correctly, setting `normalize_ae` to 1 should have the same behavior as setting `normalize_ae_gzip` to 1.